go-controller/pkg/ovn/policy_test.go

```diff
@@ -1241,16 +1231,18 @@ var _ = Describe("OVN NetworkPolicy Operations", func() {
 					context.TODO(), namespace1.Name, metav1.GetOptions{})
 				Expect(err).NotTo(HaveOccurred())
 				Expect(ns).NotTo(BeNil())
-				Eventually(fExec.CalledMatchesExpected).Should(BeTrue(), fExec.ErrorDesc)
+
+				ns.Annotations[nsMulticastAnnotation] = "true"
```

Comment:
@vishnoianil this block that calls Update() needs to be *after* the fakeExec commands are added. Because the Update() could trigger ovn-nbctl calls before the fakeExec stuff is in-place.

Reply:
Basically, I think you should revert this hunk of the diff

Reply:
Plus you need to make sure WatchNamespaces() is before WatchPods()

Reply:
eg, see https://github.com/ovn-org/ovn-kubernetes/pull/1757/files#diff-2a5ea421d13f2f639ad217cada1f5c741e9ba6d68abcf431a700c0f68b71e04dR1224

Reply:
Yeah, I already tested this. Tests passes if i just move the watchNamespace before watchPods, so i believe moving adding commands before the update() has no impact. But anyways doesn't matter to revert it. @trozet what are your thoughts here? just don't want to go back and forth on this :).

Reply:
I'm going to strenuously disagree with Tim here; we want https://github.com/ovn-org/ovn-kubernetes/pull/1643 to satisfy both me and Tim, then we always call Namespace before Pod and we're done with it.